libraries/AP_HAL_ChibiOS/hwdef/revo-mini-sd/hwdef.dat

```diff
@@ -0,0 +1,20 @@
+# Revo-mini target with I²C and SD card support 
+
+include ../revo-mini-i2c/hwdef.dat
+
+PA0 UART4_TX UART4 ALT(6)
+PA1 UART4_RX UART4 ALT(6)
+
+PA15 SDCARD_CS CS
+
+SPIDEV sdcard    SPI3 DEVID2 SDCARD_CS MODE0 400*KHZ 25*MHZ
+
+# disable logging to dataflash
+undef HAL_LOGGING_DATAFLASH_ENABLED
```

Comment:
You should also undef the pin definitions for the SPI device

Reply:
I thought people could still want to use the dataflash chip but indeed with the SD card no real need

Reply:
How to "undef" the `SPIDEV dataflash` line ?

Reply:
The SD card also uses SPI3. The only pin I could undefine is DATAFLASH_CS but I can't because of the dataflash SPIDEV definition. I think it is fine to leave the way it is now